Files/Views/SettingsPages/Experimental.xaml

```diff
@@ -45,7 +46,18 @@
                     Header="Cache files and folders for better performance"
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsOn="{Binding UseFileListCache, Mode=TwoWay}" />
+
+                <controls:NumberBox 
+                    x:Uid="PreemptiveCacheParallelLimits"
+                    HorizontalAlignment="Left"
+                    Header="Preemptive cache parallel limit (0 turns off the preemptive cache)" 
+                    SpinButtonPlacementMode="Inline"
+                    Width="450"
+                    Value="{Binding PreemptiveCacheParallelLimit, Mode=TwoWay}" 
+                    Minimum="0" 
+                    HeaderTemplate="{StaticResource CustomHeaderStyle}"
+                    IsEnabled="{Binding UseFileListCache, Mode=OneWay}"/>
```

Comment:
Would you be able to create a page on the [documentation ](https://github.com/files-community/files-community.github.io/blob/main/docs) site explaining how this works?

Reply:
Sure, i'll create performance article there

Reply:
https://github.com/files-community/files-community.github.io/pull/23